src/HttpHdrScTarget.h

```diff
@@ -10,8 +10,10 @@
 #define SQUID_HTTPHDRSURROGATECONTROLTARGET_H
 
 #include "defines.h" //for bit mask operations
-#include "HttpHdrSc.h"
+#include "http/forward.h"
+#include "SquidString.h"
 
+class HttpHdrSc;
```

Comment:
If possible, please move this forward declaration to `http/forward.h`. IMO, it is OK that this class is not in `src/http/` yet.

Reply:
Done in 46054e7